data/json/itemgroups/SUS/evac_shelter.json

```diff
@@ -56,6 +57,17 @@
           { "item": "bowl_plastic", "prob": 50 }
         ],
         "prob": 90
+      },
+      {
+        "collection": [
+          { "item": "mask_dust", "prob": 70 },
+          { "item": "whistle", "prob": 60 },
+          { "item": "mirror", "prob": 20 },
+          { "item": "pliers", "prob": 15 },
+          { "item": "candle", "prob": 15 },
+          { "item": "vitamins", "prob": 10 }
+        ],
+        "prob": 20
```

Comment:
This doesn't make any sense to me. Why would these items be appearing together, but only in lockers that have been used by other evacuees? Did you mean to put this elsewhere? Or did you not mean to make this a collection?

Reply:
I wasn't entirely sure what the intent of SUS, and so to hedge my bets I figured better to put new content in used lockers.

Reply:
So currently, this means there is a 20% chance of the list you made all appearing together. When you call them a collection, it means they are supposed to appear together all at once. It's for things like a box that contains a bunch of spoons and forks and knives, and if that box appears you want to have spoons and forks and knives all appearing together in it. Since these items don't appear to be related to each other they don't need to be in a collection together.

Reply:
Erk, yeah that collection's theme was "less likely items," "new stuff
Jerimee just added," or "things more likely to have been looted already." 

How would you like it to be changed?

Reply:
Oh I see what you mean, I will change it so it is a distribution.

Reply:
These aren't related to each other! Don't put them into a group together. This setup means if there are pliers, there will never be a whistle, and that also doesn't make sense. Put these items into existing collections where it makes sense for them to appear with the other stuff in the collection.